tests/kafkatest/services/kafka/kafka.py

```diff
@@ -42,15 +42,19 @@ class KafkaService(JmxMixin, Service):
     LOG4J_CONFIG = os.path.join(PERSISTENT_ROOT, "kafka-log4j.properties")
     # Logs such as controller.log, server.log, etc all go here
     OPERATIONAL_LOG_DIR = os.path.join(PERSISTENT_ROOT, "kafka-operational-logs")
+    OPERATIONAL_LOG_INFO_DIR = os.path.join(OPERATIONAL_LOG_DIR, "info")
+    OPERATIONAL_LOG_DEBUG_DIR = os.path.join(OPERATIONAL_LOG_DIR, "debug")
     # Kafka log segments etc go here
     DATA_LOG_DIR = os.path.join(PERSISTENT_ROOT, "kafka-data-logs")
     CONFIG_FILE = os.path.join(PERSISTENT_ROOT, "kafka.properties")
 
-
     logs = {
-        "kafka_operational_logs": {
-            "path": OPERATIONAL_LOG_DIR,
+        "kafka_operational_logs_info": {
```

Comment:
More general duckape thought about how this works -- should we always make these constants in the class? The way we use them now, I think this constant has to be copied into any test which wants to mark a specific log for collection, e.g., in the case of failure where it knows which files will be relevant for debugging. This seems brittle given changes like this. Do we know if the code that marks logs for collection will even fail if there had been tests that marked `"kafka_operational_logs"` for collection?